### PR TITLE
Release notes 1.20 and 1.20.1

### PR DESCRIPTION
**Pull request by LesiaPavlenko**
Updated version of release notes for v.1.20 and v.1.20.1.

### CODE REVIEW

File: help/deploy/releases/release-history.md [hunk 1]
```diff
@@ -20,184 +20,159 @@ position: 6 # float position is supported
...
+### Visualization and usability improvements:
+
+* MlFlow registry integration.
+* Forms autosize implementation.
+* Lightweight Predictive modeling improvements.
+* Layout editing harmonization for Data Query.
+* Macromolecules updates.
+* Diff Studio: implementing the debugging mode.
+* Dragging an entity into a project restyling.
+* Adding the 'Add to favorites' command to objects.
```
Comment (skalkin) [inline help/deploy/releases/release-history.md:56]: Should be simplified

File: help/deploy/releases/release-history.md [hunk 2]
```diff
@@ -20,184 +20,159 @@ position: 6 # float position is supported
...
+  * GROK-15887: Viewer window can't be resized in height when it's too high on the screen. 
+  * GROK-16234: Wrong label shortening. 
+  * GROK-16331: Molecules are rendered incorrectly on resizing bar chart. 
+  
+### Box plot
+* Improvements:
+  * [#2905](https://github.com/datagrok-ai/public/issues/2905): Add ''Size by" or "Shape by" or "color by'' options. 
+  * GROK-9748: Violin-state option. 
+* Fix:
+  * GROK-16244: Violin style: category bug.
```
Comment (skalkin) [inline help/deploy/releases/release-history.md:115]: If we introduce a feature in this release, why list the fixed bug related to that feature?

File: help/deploy/releases/release-history.md [hunk 3]
```diff
@@ -20,184 +20,159 @@ position: 6 # float position is supported
...
+  * GROK-15793: Calculate Identity, Similarity throw Index out of bounds.
 
+### [Chem](https://github.com/datagrok-ai/public/blob/master/packages/Chem/CHANGELOG.md)
+* Fixes:
+  * GROK-15299: Functions can't be saved to a project with data sync. 
+  * GROK-15790: Transform to smiles doesn't change units. 
+ 
+### Scripting
+* Improvements and fixes:
+  * GROK-15424: Cascade parametrized inputs in scripts.
```
Comment (skalkin) [inline help/deploy/releases/release-history.md:162]: Would be nice to link to help articles that explain what the feature is (of course, not everything is documented yet - but for instance this feature should be)

**Review by skalkin — CHANGES_REQUESTED** [review]
This is definitely a big improvement! But let's address the issues I've identified, at least most of them

File: help/deploy/releases/release-history.md [hunk 4]
```diff
@@ -20,184 +20,159 @@ position: 6 # float position is supported
...
-### Addressed Issues
+### Breaking changes:
+ 
+* File caching updates.
+* Inputs harmonization.
+ 
+### Visualization and usability improvements:
+
+* MlFlow registry integration.
+* Forms autosize implementation.
```
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:50]: changed

File: help/deploy/releases/release-history.md [hunk 5]
```diff
@@ -20,184 +20,159 @@ position: 6 # float position is supported
...
+* Lightweight Predictive modeling improvements.
```
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:51]: changed to p

File: help/deploy/releases/release-history.md [hunk 6]
```diff
@@ -20,184 +20,159 @@ position: 6 # float position is supported
...
+* Layout editing harmonization for Data Query.
+* Macromolecules updates.
+* Diff Studio: implementing the debugging mode.
+* Dragging an entity into a project restyling.
```
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:55]: Restyling for dragging an entity into a project - is a visualization improvement that makes dragging process move intuitively clear. 
This why I didn't remove this update.

[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:55]: renamed to 'Restyled the process of dragging an entity into a project'
this variant is more clear

File: help/deploy/releases/release-history.md (hunk 1 above)
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:56]: changed to 'Objects can be added to the favorites list.'

File: help/deploy/releases/release-history.md (hunk 2 above)
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:115]: removed bug mentioning

File: help/deploy/releases/release-history.md [hunk 7]
```diff
@@ -20,184 +20,159 @@ position: 6 # float position is supported
...
+* File caching updates.
+* Inputs harmonization.
+ 
+### Visualization and usability improvements:
+
+* MlFlow registry integration.
+* Forms autosize implementation.
+* Lightweight Predictive modeling improvements.
+* Layout editing harmonization for Data Query.
+* Macromolecules updates.
```
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:53]: Added additional description after it's review with Alexandr Tanas

Macromolecules updates:
- long sequences visualization optimized
- cell renderer per type for small molecules and column width changes
- monomer tooltip polished
- monomer placer optimization for column width changed
- work with missed monomers in Helm

File: help/deploy/releases/release-history.md (hunk 3 above)
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:162]: added the link:
https://datagrok.ai/help/datagrok/concepts/functions/func-params-annotation#function-inputs

File: help/deploy/releases/release-history.md [hunk 8]
```diff
@@ -20,184 +20,159 @@ position: 6 # float position is supported
...
-* (Bug) GROK-15211: Databases | New Run_query result creates without name 
-* (Improvement) GROK-15972: Connections | Updates for 'creating a connection' form 
-* (Bug) GROK-16325: Scripting | Usage: Usage does not display runs 
-* (Bug) GROK-16305: Core | Pivot table: Changing columns (pviot, group by or aggregate) through property panel throws error 
-* (Bug) [#2927](https://github.com/datagrok-ai/public/issues/2927): #2927: Heatmap is in a collapsed state after applying saved layout if it was stacked with another viewer 
-* GROK-16239: #2927: Heatmap is in a collapsed state after applying saved layout if it was stacked with another viewer 
-
-
-## 2024-07-23 1.20.0
+## 2024-07-23 Datagrok 1.20.0 release
```
[the remark this answers is not in the document]
Comment (LesiaPavlenko) [reply help/deploy/releases/release-history.md:40]: Added this part:

The Datagrok 1.20.0 release focuses on improving stability, performance, and usability, introducing refined workflows to make the platform more intuitive and reliable. Some of the key improvements in this release include:

* Lightweight predictive modeling improvements. [MlFlow registry integration](https://youtu.be/RS163zKe7s8?t=1424).
* File caching feature allows users to significantly improve the performance and efficiency of their storage operations. It reduces the time required to access frequently used files by storing them locally in browser cache. This feature is compatible with various file storages, including Amazon S3, Azure Blob Storage, and more. Users can customize expiration policies for individual files and folders to suit their specific needs. Also, caching significantly lower storage costs by reducing number of requests (follow our [guideline](https://datagrok.ai/help/access/files/#caching-files-shares) to configure cache for your connection). 
* Inputs harmonization brought significant stability enhancements and API improvements, ensuring a smoother and more consistent experience across the platform.

**Review by skalkin — APPROVED** [review]

**Pull request merged**